CI | Nightly RPM Build - Master Branch - Fails (Arch `ppc64le`)

### Environment info
- NooBaa Version: current, 5.18.0
- Platform: CI, Github workflow

### Actual behavior
1.  Nightly RPM Build - Master Branch Fails

### Expected behavior
1.  Nightly RPM Build - Master Branch should not fail.

### Steps to reproduce
1. I saw it in our actions:
https://github.com/noobaa/noobaa-core/actions/workflows/nightly-rpm-master-build.yaml

### More information - Screenshots / Logs / Other output
When looking inside we can see that it comes from:
call-master-rpm-build-and-upload / build-rpm-centos9-ppc64le / rpm-build-and-upload-artifact

In the step: "Finalize RPM"

> 0s
> Run DATE=$(date +'%Y%m%d')
> rpm_full_path=noobaa-core-5.[1](https://github.com/noobaa/noobaa-core/actions/runs/12149636040/job/33880739772#step:7:1)8.0-20241204-master.el9.ppc64le.rpm
> cp: cannot stat './build/rpm/noobaa-core-5.18.0-20241204.el9.ppc64le.rpm': No such file or directory
> Error: Process completed with exit code 1

--- a/.github/workflows/nightly-rpm-master-build.yaml
+++ b/.github/workflows/nightly-rpm-master-build.yaml
@@ -1,7 +1,7 @@
 name: Nightly RPM Build - Master Branch
 on: 
   schedule:
-    - cron: '0 23 * * *'
+    - cron: '0 0 * * *'
 
 jobs:
   call-master-rpm-build-and-upload:

--- a/.github/workflows/nightly-rpm-stage-5.15.yaml
+++ b/.github/workflows/nightly-rpm-stage-5.15.yaml
@@ -1,7 +1,7 @@
 name: Nightly RPM Build - 5.15 Branch
 on: 
   schedule:
-    - cron: '0 23 * * *'
+    - cron: '0 0 * * *'
 
 jobs:
   call-stage-5-15-4-rpm-build-and-upload:

